### DISCUSSION TITLE
Project doesn't push on to the gitub

### DISCUSSION THREAD

**Discussion by ferajofficial** [community/community#29892] [Other Feature Feedback, Questions, & Ideas]
My project doesn't push on to My github repo but i'm able to create repo.whenever i push my project on to the repo that shows an error i.e " fatal: Authentication failed for 'https://github.com/ferajofficial/my_projectname.git/' ".......Anyone help me out

**Comment by verduscos** [comment]
This might be an indication that your authentication method is not working properly. I'd recommend looking into setting up personal access token. You can read the docs [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).

**Reply by ferajofficial** [reply]
thanks buddy👍

**Reply by ferajofficial** [reply]
@verduscos still the problem persists

**Reply by verduscos** [reply]
@ferajofficial Did you make sure to use your new personal access token via the command line before pushing?

**Reply by ferajofficial** [reply]
@verduscos 
yup..

**Comment by jfullstackdev** [comment]
uninstall Git, then try GitHub Desktop (this should work even without Git), just to know whether you can really push, if that is working then it might be your Git or any other local setup that interferes, or your PAT

**Reply by ferajofficial** [reply]
okay,Thanks

**Comment by airtower-luna** [comment]
It's possible you have invalid (expired, maybe) authentication data cached somewhere. How to check and clear that depends on your OS and (if applicable) desktop environment, though.

In general, you're much more likely to get a helpful response if you provide more details, like the full commands you use and their output.

**Reply by ferajofficial** [reply]
Will provide soon !

**Reply by ferajofficial** [reply]
<img width="941" alt="Screenshot 1944-05-28 at 1 40 26 PM" src="https://user-images.githubusercontent.com/110988843/185574607-8239621c-5d38-4745-9180-5571ecd63383.png">

**Reply by airtower-luna** [reply]
Now this is interesting, the URL your pushes use is not the one you tried to set as the origin remote. And the very first command explains it: When you called `git init` there was already a Git repository in the directory, and of course Git didn't destroy that. That's why adding the `origin` remote failed: it already existed, evidently with a different URL.

Now the question is: Why was there already a repository, and what are you actually trying to achieve with it? :sweat_smile: How to fix things depends on the answer.

**Reply by verduscos** [reply]
Ohh interesting.

**Comment by queenofcorgis** [comment]
@verduscos checking in here - is the problem still persisting or were one of the fine folks above able to assist?
